.github/workflows/run-playwright-tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
```

Comment:
Please add hook on release as well. Make this start before the actual release happens so that we ensure we don't push the faulty build